packages/share_plus_linux/lib/share_plus_linux.dart

```diff
@@ -15,8 +15,17 @@ class ShareLinux extends SharePlatform {
     String? subject,
     Rect? sharePositionOrigin,
   }) {
-    final uri = Uri.encodeFull('mailto:?subject=$subject&body=$text');
-    return launch(uri);
+    final queryParameters = {
+      if (subject != null) 'subject': subject,
+      'body': text,
+    };
+
+    var uri = Uri(
+      scheme: 'mailto',
+      queryParameters: queryParameters,
+    );
+
+    return launch(uri.toString());
```

Comment:
We're calling launch unconditionally. That should probably be guarded by a call to `canLaunch`. Or is there something I'm missing?

Reply:
I added a `canLaunch` check.